### ISSUE TITLE
App crashes with python trace

### ISSUE THREAD

**Issue by eda2z** [realmazharhussain/gdm-settings#118]
### What happened?

Recently the app is terminating with an error. If I run it by clicking on its icon it hangs for several seconds but nothing appears. If I run it from the command line I get the following:

[ed@spike ~]$ gdm-settings 
Traceback (most recent call last):
  File "/usr/lib/python3.10/site-packages/gdm_settings/app.py", line 104, in do_activate
    self.window = GdmSettingsWindow(self)
  File "/usr/lib/python3.10/site-packages/gdm_settings/window.py", line 79, in __init__
    self.add_pages()
  File "/usr/lib/python3.10/site-packages/gdm_settings/window.py", line 96, in add_pages
    add_page('appearance', _('Appearance'),       pages.AppearancePageContent(self))
  File "/usr/lib/python3.10/site-packages/gdm_settings/pages/appearance.py", line 41, in __init__
    self.bind_to_gsettings()
  File "/usr/lib/python3.10/site-packages/gdm_settings/pages/appearance.py", line 80, in bind_to_gsettings
    bind_comborow_by_enum(self.background_type_comborow,
  File "/usr/lib/python3.10/site-packages/gdm_settings/bind_utils.py", line 47, in bind_comborow_by_enum
    bind_by_enum(settings, key, comborow, 'selected', enum, flags)
  File "/usr/lib/python3.10/site-packages/gdm_settings/bind_utils.py", line 40, in bind_by_enum
    bind_with_mapping(settings, key, widget, property, flags,
  File "/usr/lib/python3.10/site-packages/gdm_settings/bind_utils.py", line 123, in bind_with_mapping
    key_changed(settings, key)
  File "/usr/lib/python3.10/site-packages/gdm_settings/bind_utils.py", line 95, in key_changed
    widget.set_property(prop, key_to_prop(settings[key], user_data))
  File "/usr/lib/python3.10/site-packages/gdm_settings/bind_utils.py", line 34, in enum_name_to_value
    return enum[name].value
  File "/usr/lib/python3.10/enum.py", line 440, in __getitem__
    return cls._member_map_[name]
KeyError: 'Image'

### Expected behavior

A gui screen should appear.

### To Reproduce

Either click on its icon or execute from the command line.

### Application Version

1.0-1

### Source

AUR

### Environment

OS: Arch - updated as of 9/20/2022
Gnome: 42.5

**Comment by realmazharhussain** [comment]
1. Download this [patch.txt](https://github.com/realmazharhussain/gdm-settings/files/9689601/patch.txt) file
1. Run the following command in the terminal

   ```bash
   sudo patch /usr/lib/python3.10/site-packages/gdm_settings/bind_utils.py ~/Downloads/patch.txt
   ```

1. Try and run the app again
1. Tell me here

**Comment by eda2z** [comment]
I now get the following error from the command line (clicking on icon results in a "busy" cursor):

(gdm-settings:1930): GLib-GIO-ERROR **: 09:18:12.716: Settings schema 'io.github.realmazharhussain.GdmSettings.top-bar' does not contain a key named 'disable-arrows'
Trace/breakpoint trap (core dumped)

**Comment by realmazharhussain** [comment]
It seems like you have two versions of the app installed on your system at the same time. Is that correct?

**Comment by realmazharhussain** [comment]
Try installing [gdm-settings-git](https://aur.archlinux.org/packages/gdm-settings-git) package and see if it works.

**Comment by eda2z** [comment]
I checked and could only find 1 gdm-settings executable. I uninstalled gdm-settings via pacman. I scanned the entire file system and found a number of places where directory/file names contained either gdm-settings or gdm_settings - apparently I had installed an earlier version at some point and did not completely remove it. I deleted all of these directories/files, re-installed gdm-settings and got my original error with the long trace. I applied the patch and got my second error. I finally uninstalled gdm-settings and installed gdm-settings-git and it works fine. I don't know what to tell you. The git version works fine. If you want me to look into something please let me know otherwise I'll close this issue.

**Comment by realmazharhussain** [comment]
I added a fix as commit 8440a3e409d88eabd7377bd3731e8170b95f0488. That is why `gdm-settings-git` package works now.

`gdm-settings` package would also work if you ran the command

```bash
sudo glib-compile-schemas <prefix>/share/glib-2.0/schemas
```

Just replace `<prefix>` with the prefix where earlier version of gdm-settings was installed (most probably `/usr/local`).
